### ISSUE TITLE
Imported type library object recorded at a wrong address for external symbols

### ISSUE THREAD

**Issue by xusheng6** [Vector35/binaryninja-api#4905]
Turn on debug log, and open an PE file, we see that for each import type library object, they are registered twice:

1. at the IAT address )e.g., 0x140002000). This one is correct, and it is caused by the definition of `ImportAddressSymbol`
2. at the start of the view (0x140000000). This one's address is wrong, and it is caused by the definition of `ExternalSymbol`

```
[Analysis.TypeLibraryMapping] Recorded imported object ntoskrnl.exe!RtlInitUnicodeString at windows-kernel-x86_64/140002000
[Analysis.TypeLibraryMapping] Recorded imported object ntoskrnl.exe!RtlInitUnicodeString at windows-kernel-x86_64/140000000
[Analysis.TypeLibraryMapping] Recorded imported object ntoskrnl.exe!DbgPrintEx at windows-kernel-x86_64/140002008
[Analysis.TypeLibraryMapping] Recorded imported object ntoskrnl.exe!DbgPrintEx at windows-kernel-x86_64/140000000
[Analysis.TypeLibraryMapping] Recorded imported object ntoskrnl.exe!IofCompleteRequest at windows-kernel-x86_64/140002010
[Analysis.TypeLibraryMapping] Recorded imported object ntoskrnl.exe!IofCompleteRequest at windows-kernel-x86_64/140000000
[Analysis.TypeLibraryMapping] Recorded imported object ntoskrnl.exe!IoCreateDevice at windows-kernel-x86_64/140002018
[Analysis.TypeLibraryMapping] Recorded imported object ntoskrnl.exe!IoCreateDevice at windows-kernel-x86_64/140000000
...
```

The relevant code is at https://github.com/Vector35/view-pe/blob/6e4448fbe2ad88121db79ce36ac07c994d1184cd/peview.cpp#L1540-L1541.

The problem is, for `ExternalSymbol`, their address is unknown when they get defined (the actual address is determined during view finalization). However, the code in `PEView::AddPESymbol` is unaware of this, and defines the symbol at the start of the view, rather than the actual address of the external symbol.

**Comment by xusheng6** [comment]
I found this bug while working on https://github.com/Vector35/binaryninja-api/issues/4883

**Comment by plafosse** [comment]
Fixed in 3.6.4817
